.github/workflows/ci-core.yml

```diff
@@ -400,7 +421,8 @@ jobs:
 
   clean:
     name: Clean Go Tidy & Generate
-    if: ${{ !contains(join(github.event.pull_request.labels.*.name, ' '), 'skip-smoke-tests') && github.actor != 'dependabot[bot]' }}
+    needs: [filter]
+    if: ${{ !contains(join(github.event.pull_request.labels.*.name, ' '), 'skip-smoke-tests') && github.actor != 'dependabot[bot]' && needs.filter.outputs.not_ignored_changes == 'true' }}
```

Comment:
This applies to all modules. It must not be skipped for `integration-tests/` changes.

Reply:
Will be fixed in 29eba8e0bcf625b17870e36793b80925416b9aeb

Reply:
@jmank88 fixed in 29eba8e0bcf625b17870e36793b80925416b9aeb.